examples/beam/src/main/java/org/apache/nemo/examples/beam/tpch/Schemas.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nemo.examples.beam.tpch;
+
+import org.apache.beam.sdk.schemas.Schema;
+
+/**
+ * TPC-H Schemas.
+ * (Copied and adapted from https://github.com/apache/beam/pull/6240)
```

Comment:
Can we have file path rather than link to the PR?
(I think 'GitHub' URLs cannot be canonical way to represent resources by other ASF projects.)